addons/pause/addon.json

```
@@ -1,6 +1,12 @@
 {
   "name": "Pause button",
   "description": "Adds a pause button next to the green flag.",
+  "info": [
+    {
+      "text": "The project can also be paused with Alt+X.",
```

Had to Google whether Macs have a key named "Alt", apparently not.

```suggestion
      "text": "The project can also be paused with Alt+X (Option+X on macOS).",
```